.circleci/docker/Dockerfile.emscripten

```diff
@@ -47,6 +45,7 @@ RUN set -ex; \
 		-DZ3_BUILD_TEST_EXECUTABLES=OFF \
 		-DZ3_BUILD_EXECUTABLE=OFF \
 		-DZ3_SINGLE_THREADED=ON \
+		-DCMAKE_CXX_FLAGS="-s DISABLE_EXCEPTION_CATCHING=0" \
```

Comment:
We should think about which other flags we might want to take over from our settings to here.